modules/library/render/src/test/java/org/geotools/renderer/lite/LetterConflictTest.java

```diff
@@ -0,0 +1,265 @@
+package org.geotools.renderer.lite;
```

Comment:
I will need to ask you to include the appropriate GeoTools header when added new files.